README.md

```diff
@@ -37,322 +37,137 @@ A service that:
-> ```
-> docker compose build
-> ```
->
-> After the image is built, run `docker compose up` per item 5 above.
-
-### Step 2: Installing Python dependencies
-
-To interact with the nv-ingest service, you can do so from the host, or by `docker exec`-ing into the nv-ingest container.
-
-To interact from the host, you'll need a Python environment and install the client dependencies:
-```bash
-# conda not required, but makes it easy to create a fresh python environment
-conda create --name nv-ingest-dev --file ./conda/environments/nv_ingest_environment.yml
-conda activate nv-ingest-dev
-
-cd client
-pip install .
-
-# When not using Conda, pip dependencies for the client can be installed directly via pip. Pip based installation of
-# the ingest service is not supported.
-cd client
-pip install -r requirements.txt
-pip install .
+3. [Obtain an API key from build.nvidia.com](https://build.nvidia.com/nvidia/llama-3_2-nv-embedqa-1b-v2?snippet_tab=Python&signin=true&api_key=true)
+4. Add your key to .env
+```.env
+NVIDIA_BUILD_API_KEY=...
 ```
 
-> [!NOTE]
-> Interacting from the host depends on the appropriate port being exposed from the nv-ingest container to the host as defined in [docker-compose.yaml](docker-compose.yaml).
->
-> If you prefer, you can disable exposing that port, and interact with the nv-ingest service directly from within its container.
->
-> To interact within the container:
-> ```
-> docker exec -it nv-ingest-nv-ingest-ms-runtime-1 bash
-> ```
-> You'll be in the `/workspace` directory, which has `DATASET_ROOT` from the .env file mounted at `./data`. The pre-activated `morpheus` conda environment has all the python client libraries pre-installed:
-> ```
-> (morpheus) root@aba77e2a4bde:/workspace#
-> ```
->
-> From the bash prompt above, you can run nv-ingest-cli and Python examples described below.
-
-### Step 3: Ingesting Documents
+Note: To use library mode with self hosted NIMs: TODO
 
-You can submit jobs programmatically in Python or via the nv-ingest-cli tool.
-
-In the below examples, we are doing text, chart, table, and image extraction:
-- `extract_text`, - uses [PDFium](https://github.com/pypdfium2-team/pypdfium2/) to find and extract text from pages
-- `extract_images` - uses [PDFium](https://github.com/pypdfium2-team/pypdfium2/) to extract images
-- `extract_tables` - uses [YOLOX](https://github.com/Megvii-BaseDetection/YOLOX) to find tables and charts. Uses [PaddleOCR](https://github.com/PaddlePaddle/PaddleOCR) for table extraction, and [Deplot](https://huggingface.co/google/deplot) and CACHED for chart extraction
-- `extract_charts` - (optional) enables or disables the use of Deplot and CACHED for chart extraction.
-
-> [!IMPORTANT]
-> `extract_tables` controls extraction for both tables and charts. You can optionally disable chart extraction by setting `extract_charts` to false.
+5. Start ingesting and retrieving documents:
```

Comment:
Generally seems fine, although I do think having the associated CLI command would be a good idea for anyone who doesn't wants a code-free approach.